### PR TITLE
Update dependency versions

### PR DESCRIPTION
**Pull request by mgkwill**
Signed-off-by: Marcus G K Williams <Marcus G K Williams 168222+mgkwill@users.noreply.github.com>

Issue Number: #85

Objective of pull request: Update dependency versions to latest specified by @bamsumit 

## Pull request checklist

Your PR fulfills the following requirements:
- [ ] [Issue](https://github.com/lava-nc/lava-dl/issues) created that explains the change and why it's needed
- [ ] Tests are part of the PR (for bug fixes / features)
- [ ] [Docs](https://github.com/lava-nc/docs) reviewed and added / updated if needed (for bug fixes / features)
- [ ] PR conforms to [Coding Conventions](https://lava-nc.org/developer_guide.html#coding-conventions)
- [ ] [PR applys BSD 3-clause or LGPL2.1+ Licenses](https://lava-nc.org/developer_guide.html#add-a-license) to all code files
- [ ] Lint (`flakeheaven lint src/lava tests/`) and (`bandit -r src/lava/.`) pass locally
- [ ] Build tests (`pytest`) passes locally

## Pull request type

 

 

Please check your PR type:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation changes
- [ ] Other (please describe): 

## What is the current behavior?

-

## What is the new behavior?

-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -45,17 +45,17 @@ classifiers = [
...
-torch = "1.8.1"
-torchvision = "0.9.1"
-h5py = "^3.1.0"
-ninja = "^1.10.2"
-matplotlib = "^3.5.1"
-numpy = "^1.22.2"
-scipy = "^1.8.0"
-pillow = "^9.0.1"
-pytest = "^7.0.1"
+lava-nc = { git = "https://github.com/lava-nc/lava.git", branch = "main", tag = "v0.4.0" }
```
Comment (bamsumit) [inline pyproject.toml:48]: Can we change it to point to the latest version of lava? There are some fixes I am issuing that are needed to fix the broken changes in lava 0.4.0

**Review by bamsumit — APPROVED** [review]
Looks good Marcus.

Just have one suggestion to point to the latest lava instead of tag 0.4.0.

The CI are broken at the current stage. I will create a PR to fix it soon.

File: pyproject.toml (hunk 1 above)
Comment (mgkwill) [reply pyproject.toml:48]: ACK

**Comment by mgkwill** [comment]
Closing, as the versions have been updated already.

**Pull request closed without merging**